Core/Base/DB.py

```diff
@@ -51,5 +75,27 @@ def __init__(self, dbname, fullname, debug=False):
 
 #############################################################################
   def getCSOption(self, optionName, defaultValue=None):
+    """ Get option from CS
+
+        :param str optionName: option name
+        :param defaultValue: default value
+
+        :return: value that inherits the defaultValue type
+    """
     cs_path = getDatabaseSection(self.fullname)
     return gConfig.getValue("/%s/%s" % (cs_path, optionName), defaultValue)
+
+  def updateDBVersion(self, version):
```

Comment:
Right now this is not called anywhere. Which component should be its client?

Reply:
This will be used in future PR for ProxyDB to change the tables